src/west/manifest.py

```diff
@@ -1352,6 +1360,8 @@ def __init__(self, *,  # All arguments are keyword-only.
         # These back the relevant properties.
         self._posixpath: Optional[str] = None
         self._repo_posixpath: Optional[str] = None
+        # This backs the userdata property
```

Comment:
I think you can just say `self.userdata: Any = None` here, and update `self.userdata` in `_load_self` as needed. There's no need for a property. Also, where is the property definition? I don't see it in this patch.

Reply:
I believe I have addressed your suggestion here.